Yuneec_SDK_iOS/YNCSDKCamera.h

```diff
@@ -24,6 +24,37 @@ typedef void (^YNCCameraCompletion)(NSError *error);
  */
 typedef void (^YNCReceiveDataCompletionBlock)(YNCCameraSettings *receiveData, NSError *error);
 
+/**
+ Data type for the different color modes.
+ */
+typedef NS_ENUM (NSInteger, YNCCameraColorMode) {
+    /**
+     Neutral mode
+     */
+    YNCNEUTRAL = 0,
+    /**
+     Enhanced mode
+     */
+    YNCENHANCED,
+    /**
+     Night mode
+     */
+    YNCNIGHT,
+    /**
+     Unprocesses mode
```

Comment:
We should document what `Unprocessed` means. It means that no additional color processing has been performed and no artificial sharpening.